backend/scripts/rollback.sh

```
@@ -0,0 +1,99 @@
+fi
+
+# Restore MySQL database before updating symlink
+echo "Restoring MySQL database to previous state..."
+restore_mysql_database
+
+# Update current symlink to previous release
+ln -sfn $PREV_RELEASE ~/fin-invoice-ocr-team6/current
+echo "Updated current symlink to: $(readlink -f ~/fin-invoice-ocr-team6/current)"
+
+# Restart application with delete + start approach (same as deployment)
+cd ~/fin-invoice-ocr-team6/current/backend
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)